unittests/allplatformstests.py

```diff
@@ -54,6 +54,7 @@
 
 from mesonbuild.dependencies import PkgConfigDependency
 from mesonbuild.build import Target, ConfigurationData, Executable, SharedLibrary, StaticLibrary
+from mesonbuild.mtest import decode, discouraged_unichrs
```

Comment:
"decode" is a pretty ambiguous name to import from another module I guess. It might be better to do `from mesonbuild import mtest` and refer to mtest.decode